src/components/containers/TraceAccordion.js

```diff
@@ -12,11 +12,23 @@ const TraceFold = connectTraceToPlot(PlotlyFold);
 class TraceAccordion extends Component {
   render() {
     const {data = [], localize: _} = this.context;
-    const {canAdd, canGroup, children, messageIfEmptyFold} = this.props;
+    const {canAdd, canGroup, children, messageIfEmptyFold, panel} = this.props;
```

Comment:
I would recommend changing `panel` to `excludeFits` or something as a boolean, just to match `canAdd` and `canGroup`